Description: Fix dupes in buildd configuration
Author: Christoph Biedl <debian.axhn@manchmal.in-ulm.de>
Forwarded: yes
Last-Update: 2022-09-28

    The files

        /etc/schroot/buildd/copyfiles
        /etc/schroot/buildd/nssdatabases

    contain duplicated information.

    This was possibly introduced in 2011, in commit d7dad336 ("etc:
    Add buildd profile-template from sbuild").

--- a/etc/profile-templates/buildd/all/copyfiles
+++ b/etc/profile-templates/buildd/all/copyfiles
@@ -1,6 +1,2 @@
-# Files to copy into the chroot from the host system.
-#
-# <source-and-destination>[ <destination>]
 /etc/hosts
-/etc/resolv.conf
 /etc/networks
--- a/etc/profile-templates/buildd/all/nssdatabases
+++ /dev/null
@@ -1,7 +0,0 @@
-# System databases to copy into the chroot from the host system.
-#
-# <database name>
-passwd
-shadow
-group
-gshadow
